### PR TITLE
App config

### LINKED ISSUE #14: Use Azure App Configuration to be able to configure where images are saved at runtime
- https://search.maven.org/search?q=g:com.azure
- azure-data-appconfiguration
- Documentation: https://github.com/Azure/azure-sdk-for-java/tree/master/appconfiguration/client/README.md

## Requirements
1. Create an app configuration store in Azure
1. Create a setting called "IMAGE_STORAGE_TYPE"
   1. Those values can be either "Local" or "BlobStorage"
1. When your program starts up, it reads this configuration value from App Configuration
1. If "Local", use the local file directory to resolve any image paths
1. if "BlobStorage", print out a not supported yet and use Local.

### PR DESCRIPTION
**Pull request by Intern426**
Added App Configuration
Fixed #14

### CODE REVIEW

File: src/main/java/com/azure/app/App.java [hunk 1]
```diff
@@ -31,6 +36,23 @@
      * @param args Arguments to the library program.
      */
     public static void main(String[] args) {
+        ConfigurationAsyncClient client;
+        ConfigurationSetting setting = null;
+        try {
+            client = ConfigurationAsyncClient.builder()
```
Comment (conniey) [inline src/main/java/com/azure/app/App.java]: If the env variable AZURE_APPCONFIG is not set (that is, the value retruend from System.getEnv is null or empty), I'd print an error message telling the user what env variable to set and exit. It's easier to consume than an "IllegalArgumentException" from ConfigurationClientCredentials.

Comment (Intern426) [reply src/main/java/com/azure/app/App.java]: Wouldn't that be the same as just giving them the connection string?

Comment (conniey) [reply src/main/java/com/azure/app/App.java]: I was thinking:
```java
String connectionString = System.getEnv("AZURE_APPCONFIG");
if (connectionString == null || connectionString.equals("")) {
   System.err.println("environment variable AZURE_APPCONFIG is not set. Cannot connect to App Configuration. Please set it.");
   return -1;
}

client = ConfigurationAsyncClient.builder().credentials(new Credentials(connectionString).build();
```

Comment (Intern426) [reply src/main/java/com/azure/app/App.java]: Oh! Got it! Okay, thanks

Comment (Intern426) [reply src/main/java/com/azure/app/App.java]: Oh and if that were the case, does the program terminate altogether?

Comment (conniey) [reply src/main/java/com/azure/app/App.java]: I would terminate the program. It's a configuration error.

**Pull request merged**